javascript/eslintrc.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
     "no-shadow-restricted-names": 2,
     "no-undef": 2,
     "no-use-before-define": 2,
+    "quotes": [2, "double", "avoid-escape"],
```

Comment:
hehe github refreshes. :p

@wfleming that was my reading as well. I think that rule only applies to es6 though, and I don't think we're using that on most of our projects.

I tried adding
```
var test = `test` 
```
to a file and analyzing with double quotes rule in place, but no issue was found.